test/blackbox-tests/test-cases/coq/failed-config.t/run.t

```diff
@@ -104,22 +104,16 @@ Here we query the version of Coq. Due to the expansion of %{coq:_} macros we nee
 Fails for now, but could be improved in the future.
```

Comment:
Has improved now, we can update the comment. We can say something like: Querying the Coq version is not affected by the inability to query the Coq configuration.

Reply:
done